web/src/main/java/no/nav/foreldrepenger/tilbakekreving/web/app/tjenester/behandling/dto/BehandlingDtoTjeneste.java

```diff
@@ -294,10 +295,24 @@ private void leggTilLenkerForBehandlingsoperasjoner(BehandlingDto dto) {
         dto.leggTil(post(AksjonspunktRestTjeneste.AKSJONSPUNKT_PATH, "lagre-aksjonspunkter"));
 
         dto.leggTil(post(ForeldelseRestTjeneste.BASE_PATH + "/belop", "beregne-feilutbetalt-belop"));
-        if (!BehandlingStatus.AVSLUTTET.equals(dto.getStatus()) && !dto.isBehandlingPåVent()) {
-            dto.leggTil(post(VergeRestTjeneste.BASE_PATH + "/opprett", "opprett-verge"));
-            dto.leggTil(post(VergeRestTjeneste.BASE_PATH + "/fjern", "fjern-verge"));
+        var finnesVerge = vergeRepository.finnesVerge(behandling.getId());
+        var tillatteoperasjoner = lovligeOperasjoner(behandling, finnesVerge);
```

Comment:
Du kan vurdere å løfte sjekk opp et par hakk og beholde gammel logikk for k9 og ny for fp
Da blir det enklere å rydde ved evt repo-fork.

Reply:
som i if(k9) - kode som før (if !avsluttet && !påvent) legg til else ny kode.